docs/api-gateway/03-04-migration.md

```diff
@@ -31,12 +26,26 @@ kubectl delete policy -n ${API_NAMESPACE} ${API_NAME} --ignore-not-found
 # recreate Api
 kubectl get api -n ${API_NAMESPACE} ${API_NAME} -o yaml | kubectl replace --force -f 
 ```
-Once the Apis are recreated, check again if the status is present. The output should not include any Api resources, allowing you to proceed with the migration. Follow these steps to ensure your services are properly migrated:
+3. Once the Apis are recreated, check again if the status is present.
+
+## Upgrade
+
+If all Apis have the status, proceed with the upgrade to migrate the resources.
+
+>**CAUTION:** Migrating resources may result in a temporary downtime of the exposed service. 
+
+During upgrade, it may turn out that some resource specifications are too complex or fail to meet all the migration requirements. In such a case the process skips them but doesn't break the way existing services are exposed. However, if you want to introduce further changes or remove the Api CR, your actions won't have any effect on how the service is exposed because it will still use the original configuration.  
```

Comment:
```suggestion
During the upgrade process, it may turn out that some resource specifications are too complex or fail to meet all the migration requirements. In such a case, the process skips them, but it doesn't break the way existing services are exposed. However, if you want to introduce further changes or remove the Api CR, your actions won't have any effect on how the service is exposed, because it will still use the original configuration.  
```